client/src/components/CardDialog/CardDialog.tsx

```diff
@@ -62,13 +60,13 @@ const CardDialog = ({ name = 'blank', columnId, tag = 'white', id }: DialogProps
               <Box className={`${classes.cardTag} ${colorClasses.cardTagColor}`}></Box>
             </Grid>
             <Typography variant="body2" className={classes.dialogSubTitle}>
-              {`In list "${getColNameById(columnId)}"`}
+              {`In list "${column?.name || 'None'}"`}
```

Comment:
I think this is rare to happen where a column doesn't exist however, I could rework this to not show the card detail if the column wasn't found.

Reply:
Hmmm yeah this shouldn't really happen. We should probably throw an error in this situation

Reply:
@sina-jamshidi Yeah, I'll do some error handling in my next changes 👍🏻

Reply:
@sina-jamshidi Went a head and did some checks if the column happens to be null. I highly doubt this could happen but if it does it just doesn't open the card dialog and it sends a Snackbar notification